src/layouts/Login/LoginPageSgid.tsx

```diff
@@ -0,0 +1,257 @@
+          "Your email has not been whitelisted to use sgID login. Please use another method of login."
+        break
+      default:
+        errorMessage = `Something went wrong. ${DEFAULT_RETRY_MSG}`
+    }
+    errorToast({
+      id: "sgid-login-error",
+      title: "Error",
+      description: errorMessage,
+    })
+  }, [errorToast, statusCode])
+
+  return (
+    <VStack gap="2.5rem" alignItems="start" width="65%">
+      <Text fontSize="2.5rem" color="text.title.brand" textStyle="display-2">
+        Rapidly build & launch informational sites
+      </Text>
+      <InlineMessage>
+        We’re moving in phases from GitHub IDs to email addresses as the login
+        method. For those currently using GitHub ID, you’ll be informed when you
+        can log in using the email method.
+      </InlineMessage>
+      <Tabs width="100%">
+        <TabList>
+          <Tab>Github Login</Tab>
+          <Tab>Email Login</Tab>
+          <Tab>sgID Login</Tab>
+        </TabList>
+        <TabPanels pt="2rem" minHeight="16.5rem">
+          <TabPanel>
+            <Button
+              as={Link}
+              rel="noopener noreferrer"
+              textDecoration="none"
+              _hover={{
+                textDecoration: "none",
+                bgColor: "primary.600",
+              }}
+              href={`${process.env.REACT_APP_BACKEND_URL_V2}/auth/github-redirect`}
+            >
+              <Text color="white">Log in with GitHub</Text>
+            </Button>
+          </TabPanel>
+          <TabPanel>
+            {email ? (
+              <OtpForm
+                email={email}
+                onSubmit={handleVerifyOtp}
+                onResendOtp={handleResendOtp}
+                errorMessage={
+                  getAxiosErrorMessage(loginError) ||
+                  getAxiosErrorMessage(verifyError)
+                }
+              />
+            ) : (
+              <LoginForm
+                onSubmit={handleSendOtp}
+                errorMessage={getAxiosErrorMessage(loginError)}
+              />
+            )}
+          </TabPanel>
+          <TabPanel>
+            <InlineMessage mb="1rem">
+              This is an experimental service currently offered to OGP officers
+              only.
+            </InlineMessage>
+            <Button onClick={() => getSgidAuth()} isLoading={isSgidAuthLoading}>
+              Log in with Singpass app
+            </Button>
+          </TabPanel>
+          <Text color="text.helper" fontSize="0.625rem" pt="2rem">
+            By clicking ‘Log in’, you are acknowledging and agreeing to Isomer’s{" "}
+            <Link href={TERMS_OF_USE_LINK} isExternal>
+              Terms of Use
+            </Link>
+            {" and our "}
+            <Link href={PRIVACY_POLICY_LINK} isExternal>
+              Privacy policy
+            </Link>
+          </Text>
+        </TabPanels>
+      </Tabs>
+    </VStack>
+  )
+}
+
+export const LoginPageSgid = (): JSX.Element => (
```

Comment:
this is abit confusing to me - it seems like this portion is repeated code between this/`LoginPage`? why wouldn't we just use the same layout for both?